runtime/tests/checker/interface_test.go

```diff
@@ -230,9 +233,8 @@ func TestCheckInvalidInterfaceWithInitializerImplementation(t *testing.T) {
 				),
 			)
 
-			errs := ExpectCheckerErrors(t, err, 1)
+			require.NoError(t, err)
```

Comment:
That's an interesting "edge case": Should we allow initializes and destructors as default implementations? I think we should rather exclude them, have this test keep failing.

Reply:
I agree that it makes sense to not allow default implementations for init/destructor. I don't have any concrete examples as to why it's bad, but just 'feels' it could introduce some trouble. Given that it's an addition, we can always add it in the future, if needed.